libs/design/notification/examples/src/notification-with-actions/notification-with-actions.component.html

```diff
@@ -1,14 +1,19 @@
 <button daff-button (click)="toggleNotification()">
-	<ng-container *ngIf="showNotification">Show Notification</ng-container>
-	<ng-container *ngIf="!showNotification">Hide Notification</ng-container>
+  @if (showNotification) {
```

Comment:
This is backwards. It's saying "Hide Notification" when the notification example is hidden and "Show Notification" when the button is clicked.

Reply:
I've handled this by removing the extraneous conditions.